.gitattributes

```diff
@@ -7,8 +7,10 @@
 *.pcd filter=lfs diff=lfs merge=lfs -text
 *.png filter=lfs diff=lfs merge=lfs -text
 *.record filter=lfs diff=lfs merge=lfs -text
-
 # TODO(all): Currently we have some models named as *.md which should be renamed
 # and tracked.
 *caffe.md filter=lfs diff=lfs merge=lfs -text
 *deploy.md filter=lfs diff=lfs merge=lfs -text
+cruise_cutin_vehicle_model.pt filter=lfs diff=lfs merge=lfs -text
```

Comment:
Should be some pattern that could cover all current and future such files.

Reply:
done